src/modules/extendedCallWindow/i18n/nl_NL.js

```diff
@@ -0,0 +1,140 @@
+            rw: 'Hulpverleningsvoertuig',
+            gwl2wasser: 'Slangenwagen / Watertankwagen of Watertransportsysteem-haakarmbak',
+            gwl2wasser_only: 'Alleen Slangenwagen / Watertankwagen',
+            abl2wasser_only: 'Watertransportsysteem-haakarmbak',
+            gw_taucher: 'Waterongevallenvoertuig',
+            boot: 'WOA (Water ongeval aanhanger)',
+            gwa: 'Adembeschermingsvoertuig of haakarmbak',
+            ab_atemschutz_only: 'Adembeschermingshaakarmbak',
+            gwmesstechnik: 'Dienstbus Verkenningseenheid Brandweer',
+            gwgefahrgut: 'Adviseur Gevaarlijke Stoffen',
+            mtw: 'Dienstauto / Dienstbus brandweer',
+            fustw: 'Politie Noodhulp',
+            police_motorcycle: 'Dienstmotor Noodhulp',
+            fustw_or_police_motorcycle: 'Politie Noodhulp of Dienstmotor Noodhulp',
+            polizeihubschrauber: 'Politiehelikopter',
+            hondengeleider: 'Hondengeleider',
+            ovd_p: 'Officier van Dienst - Politie',
+            lebefkw: 'ME Commandovoertuig',
+            grukw: 'ME Groepsvoertuig',
+            spokesman: 'Voorlichters',
+            at_c: 'AT-Commandant',
+            at_o: 'AT-Operator',
+            at_m: 'AT-Materiaalwagen',
+        },
+    },
+    enhancedMissingVehicles: {
+        vehicle: 'Voertuigtype',
+        missing: 'Missende voertuigen',
+        driving: 'Aanrijdend',
+        total: 'Nog nodig',
+        selected: 'Geselecteerd',
+        vehiclesByRequirement: {
+            [/^Tankautospuit(en)$/]: [0, 1, 6, 7, 8, 9, 12, 14, 15, 17, 50],
+            [/^Redvoertuig(en)$/]: [2, 18],
+            [/^Officieren(en) van Dienst - Brandweer$/]: [3, 19],
+            [/^Hulpverleningsvoertuig(en)$/]: [4, 51],
+            [/^Adembeschermingsvoertuig of haakarmbak$/]: [5, 27],
+            [/^Slangenwagen, watertankwagen of gelijkwaardige haakarmbak$/]: [10, 29, 34, 45],
+            [/^Hoofd Officier(en) van Dienst$/]: [19],
+            [/^Commandowagen(s)$/]: [31, 32],
+            [/^Verkenningseenh(eden|eid)$/]: [11],
+            [/^Adviseur(s) Gevaarlijke Stoffen$/]: [24],
+            [/^Waterongevallenvoertuig(en) \/ Oppervlaktereddingsteam(s)$/]: [33, 49, 50],
+            [/^Waterongevallenaanhanger(s)$/]: [36],
+            [/^Crashtender(s)$/]: [41, 42, 43],
+            [/^AFO\/OSC$/]: [44],
+            [/^Voorlichter(s)$/]: [56],
+            [/^Noodhulpeenhe(id|den)$/]: [22, 25, 46, 47, 48],
+            [/^Officier(en) van Dienst - Politie$/]: [35],
+            [/^Politiehelikopter$/]: [28],
+            [/^Hondengeleider(s)$/]: [47, 48],
+            [/^ME Commandovoertuig(en)$/]: [39],
+            [/^ME Groepsvoertuig(en)$/]: [40],
+            [/^AT Commandant(en)$/]: [53],
+            [/^AT Materiaalwagen(s)$/]: [55],
+            [/^AT Operator(s)$/]: [54],
+            [/^Ambulance(s)$/]: [16, 52, 57],
+        },
+    },
+    tailoredTabs: {
+        defaultTabs: [
```

Comment:
@KBOE2 Sollen da nicht eigentlich erstmal nur die standard rein die es gibt?
 Shouldn't there really only be the standard pure ones available?
![default](https://cdn.discordapp.com/attachments/746090047396118718/770416206355628052/unknown.png)

Reply:
The "Politie Noodhulp" and "Politie Overig" is a wish of many users. It's also already on planning.